### DISCUSSION TITLE
Roadmap - Integrate livewire

### DISCUSSION THREAD

**Discussion by misterdebug** [misterdebug/crud-generator-laravel#31] [Polls]
The CRUD index page is a simple table, the aim would be to add livewire to make this table a datatable with instant search etc...

**Comment by lutfiidham** [comment]
I Think datatables is enough, but with livewire is awesome!

**Reply by misterdebug** [reply]
Yes my proposal is to add a kind of datatable but custom without adding other plugins (like laravel datatable) or js plugins
I will leave the choice to use Livewire or not during the process, of course :)

**Reply by misterdebug** [reply]
Available in 1.10.6
more info : https://github.com/misterdebug/crud-generator-laravel/releases/tag/1.10.6

**Comment by misterdebug** [comment]
I'm doing some tests with Livewire 3 
![image](https://github.com/misterdebug/crud-generator-laravel/assets/23297600/fb8b0603-197f-4418-a4b4-2dae0a03be5f)

**Comment by misterdebug** [comment]
The tests are passing, it should be ready very soon :)
![image](https://github.com/misterdebug/crud-generator-laravel/assets/23297600/9f0190fe-9b46-4b83-a8b6-2436ce6be922)
